build_extra/rust/BUILD.gn

```diff
@@ -1301,11 +1301,15 @@ rust_proc_macro("serde_derive") {
 rust_rlib("serde_json") {
   edition = "2015"
   source_root = "$cargo_home/registry/src/github.com-1ecc6299db9ec823/serde_json-1.0.39/src/lib.rs"
-  features = [ "default" ]
+  features = [
+    "default",
+    "preserve_order",
```

Comment:
Is this feature implicitly enabled by the `indexmap` crate?
If we rely on it, shouldn't it be explicitly enabled in Cargo.toml?

Reply:
It's not, I'll add it to `Cargo.toml`